instaloader/instaloadercontext.py

```diff
@@ -486,15 +484,17 @@ def get_json(self, path: str, params: Dict[str, Any], host: str = 'www.instagram
                 raise ConnectionException(error_string) from err
 
     def graphql_query(self, query_hash: str, variables: Dict[str, Any],
-                      referer: Optional[str] = None, rhx_gis: Optional[str] = None) -> Dict[str, Any]:
+                      referer: Optional[str] = None) -> Dict[str, Any]:
         """
         Do a GraphQL Query.
 
         :param query_hash: Query identifying hash.
         :param variables: Variables for the Query.
         :param referer: HTTP Referer, or None.
-        :param rhx_gis: 'rhx_gis' variable as somewhere returned by Instagram, needed to 'sign' request
         :return: The server's response dictionary.
+
+        .. versionchanged:: 4.13
```

Comment:
```suggestion
        .. versionchanged:: 4.13.1
```

We will merge this into `master` rather than the upcoming minor update.

It does minimally change the API though, however in a way that is required to adapt to Instagram's changes. There is no valid reason anymore to pass `rhx_gis` parameter.